src/ui/view/dataverse/component/itemOverview/itemOverview.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC } from 'react'
+import './itemOverview.scss'
+
+type ItemOverviewProps = {
+  type: string
+  title: string
+  description: string
+}
+
+const ItemOverview: FC<ItemOverviewProps> = ({ type, title, description }): JSX.Element => (
+  <>
+    <div className="okp4-dataverse-portal-dataverse-item-overview-type">{type}</div>
+    <div className="okp4-dataverse-portal-dataverse-item-overview-title">{title}</div>
+    <div className="okp4-dataverse-portal-dataverse-item-overview-description">{description}</div>
```

Comment:
Please use the correct semantic html tags, div is too general. Also the `type` needs to be internationalized